clicktests/test.noheader.js

```diff
@@ -43,6 +43,10 @@ suite.test('Check for refresh button', function(done) {
   });
 });
 
+suite.test('Shutdown', function(done) {
```

Comment:
Isn't this the same as [below](https://github.com/FredrikNoren/ungit/pull/550/files#diff-adc25198ac2351b2aa9cc1351a87804bR51)?